tests/tkupchyn/homework_08/test_buildings.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from tkupchyn.homework_08.buildings import Building
+
+
+b1 = Building(1, 2, 2, 3)
+b2 = Building(1, 2, 2, 8, 5)
```

Comment:
do not create objects to be tested apart from tests
since this logic is executed during initialisation of tests but not during tests run